docs/decisions/0018-standarize-django-po-files.rst

```diff
@@ -0,0 +1,157 @@
+Standardize ``django.po`` and ``djangojs.po`` files
+===================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+#. The edx-platform splits its translations files to ``mako-studio.po``,
+   ``django-partial.po`` among 11 files in total to make it possible for
+   translators to focus on learner-facing text
+   and de-prioritize educator-facing text. The idea was that LMS would be
+   translated into more languages than CMS (Studio) would.
+
+#. The edx-platform uses the ``i18n_tools segment`` to split the ``django.po``
+   and ``djangojs.po`` files into smaller files. This is done with the goal of
+   making it easier for translators to translate the files.
+
+#. When pulling from Transifex or cutting a new release, the translations are
+   bundled into 2 files (``django.po`` and ``djangojs.po``).
+
+#. The `FC-0012 project <https://openedx.atlassian.net/l/cp/XGS0iCcQ>`_
+   which implements `Translation Infrastructure update OEP-58 <https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification>`_
+   is in progress which means that all translation files are going to live in
+   the new `openedx-translations Transifex project`_.
+
+#. The `openedx-translations GitHub repository`_
+   is going to sync the translations between repos and Transifex as well as
```

Comment:
> is going to sync the translations between repos and Transifex

It's not fully clear what's meant by this. It seems to imply that translations would still live in the repos. It might be good to explain the path a little:
repo source strings -> openedx-translations repo source strings -> transifex
transifex translated strings -> openedx-translations repo (pulled into repos as needed via atlas)

Reply:
re-worded in a more succinct way.